dotCMS/src/main/java/com/dotmarketing/portlets/contentlet/transform/strategy/DefaultTransformStrategy.java

```diff
@@ -0,0 +1,244 @@
+import static com.dotmarketing.portlets.contentlet.model.Contentlet.TITTLE_KEY;
+import static com.dotmarketing.portlets.contentlet.model.Contentlet.WORKING_KEY;
+import static com.dotmarketing.portlets.contentlet.transform.strategy.TransformOptions.BINARIES_AS_MAP;
+import static com.dotmarketing.portlets.contentlet.transform.strategy.TransformOptions.IDENTIFIER_AS_MAP;
+import static com.dotmarketing.portlets.contentlet.transform.strategy.TransformOptions.INC_BINARIES;
+import static com.dotmarketing.portlets.contentlet.transform.strategy.TransformOptions.INC_COMMON_PROPS;
+import static com.dotmarketing.portlets.contentlet.transform.strategy.TransformOptions.INC_CONSTANTS;
+import static com.dotmarketing.portlets.contentlet.transform.strategy.TransformOptions.INC_VERSION_INFO;
+import static com.dotmarketing.portlets.contentlet.transform.strategy.TransformOptions.LANGUAGE_AS_MAP;
+import static com.dotmarketing.portlets.contentlet.transform.strategy.TransformOptions.LANGUAGE_PROPS;
+import static com.dotmarketing.portlets.contentlet.transform.strategy.TransformOptions.USE_ALIAS;
+import static com.dotmarketing.portlets.contentlet.transform.strategy.TransformToolbox.NOT_APPLICABLE;
+import static com.dotmarketing.portlets.contentlet.transform.strategy.TransformToolbox.mapIdentifier;
+import static com.dotmarketing.portlets.contentlet.transform.strategy.TransformToolbox.mapLanguage;
+import static com.dotmarketing.portlets.htmlpageasset.business.HTMLPageAssetAPI.URL_FIELD;
+
+import com.dotcms.content.elasticsearch.constants.ESMappingConstants;
+import com.dotcms.contenttype.model.field.BinaryField;
+import com.dotcms.contenttype.model.field.ConstantField;
+import com.dotcms.contenttype.model.field.Field;
+import com.dotcms.contenttype.model.type.ContentType;
+import com.dotmarketing.beans.Host;
+import com.dotmarketing.beans.Identifier;
+import com.dotmarketing.business.APILocator;
+import com.dotmarketing.exception.DotDataException;
+import com.dotmarketing.exception.DotSecurityException;
+import com.dotmarketing.portlets.contentlet.model.Contentlet;
+import com.dotmarketing.portlets.languagesmanager.model.Language;
+import com.dotmarketing.util.Logger;
+import com.liferay.portal.model.User;
+import io.vavr.control.Try;
+import java.io.File;
+import java.io.IOException;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+import java.util.Optional;
+import java.util.Set;
+
+public class DefaultTransformStrategy extends AbstractTransformStrategy<Contentlet> {
+
+    DefaultTransformStrategy(final TransformToolbox toolBox) {
+        super(toolBox);
+    }
+
+    @Override
+    public Contentlet fromContentlet(final Contentlet contentlet) {
+        return contentlet;
+    }
+
+    @Override
+    public Map<String, Object> transform(final Contentlet contentlet, final Map<String, Object> map,
+            final Set<TransformOptions> options, User user)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Parameter 'user' is not assigned and could be declared final](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5596402)